data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+	`	Right on schedule the news returns with the severe anchorwoman. It turns out that the crewman in question had been picked up by a minor media outlet on Tarazed. He had apparently travelled many weeks before deciding to seek the media in order to avoid being shut down by Republic Intelligence if he contacted media elsewhere. The man is fairly young, and very well presented, though he has a bit of the paranoid about him that is plain to see even through a screen.`
+	branch "hai knowledge"
+		has "First Contact: Hai: offered"
+	`	A clip of alien space ships in combat with weapons quite unlike anything produced in human space is shown, you've seen things, but not this.`
+	`	It is only after this that the anchorwoman reveals that, while this was initially treated as a hoax, one of their own reporters posed a timely question to a well-respected trader and caused quite the stir when he slipped up and confirmed the existance of the aliens known as 'Hai'.`
+	`	This is, of course, ridiculous... but maybe it would be good to go investigate this for yourself. You listen in to collect the relevant details for future reference.`
+		accept	
+	label "hai knowledge"
+	`	A clip of alien space ships in combat with weapons quite unlike anything produced in human space is shown, and you recognise shield beetles hammering away at each other.`
+	`	It is only after this that the anchorwoman reveals that, while this was initially treated as a hoax, one of their own reporters posed a timely question to a well-respected trader and caused quite the stir when he slipped up and confirmed the existance of the aliens known as 'Hai'.`
+	`	You suspect it might be time to go visit your Hai friends again and see what's going on.`
+		accept
+
+
+conversation "What is going on in Hai space? (Have not met Unfettered.)"
+	`Landing in Hai space draws a much different type of attention than expected. Only recently you've begun to acclimate yourself to getting positive attention. As a war hero you're something of a minor celebrity now back in human space, but here suddenly mothers are keeping tight grips on their children and passers-by look upon you with a hint of wariness.`
+	`	You realise that the few humans who have settled here among the Hai are all wearing pink at the moment, and when you catch the eye of one lady who looks friendly you greet them to ask a few questions.`
+	choice
+		`	"Why does everyone seem so afraid?"`
+	`	She shakes her head and wrinkles her nose with a look of distaste, "Within about three days of hearing that the Hai were being splashed all over major news networks back home a whole heap of curious pirates have chanced the wormhole en mass. The local security forces on patrol at the time got blown out of the sky by a surprise attack and everyone has been a bit shaken up since."`
```

Comment:
I have noticed this comma/period thing in some other places in ES, but it flies in the face of what I've been taught and all usage I've observed elsewhere. If you have a source on this I'd appreciate it because otherwise I'm inclined to disagree.